packages/wdio-config/src/utils.ts

```diff
@@ -182,7 +181,7 @@ export function makeRelativeToCWD (files: (string | string[])[] = []): (string |
 
         returnFiles.push(file.startsWith('file:///')
             ? url.fileURLToPath(file)
-            : path.resolve(process.cwd(), file))
+            : file)
```

Comment:
Maybe this change should be reverted?

Reply:
I have tried to make minimal changes can you please review again?